### PR TITLE
Capstone Project features 

### PR DESCRIPTION
**Pull request by hafedEfheij**
For this PR, completed the following tasks:

Add index.html, index.css, and app.js files
Add GitHub actions workflow, gitignore, and linters: webhint, stylelint, and eslint
Add Bootstrap CSS and Lato font family
Add the navbar and font awesome icons to the home page
Add the hero section, add position fixed to the navbar of the home page
Add the program section to the home page
Add the speakers' section to the home page
Add about.html, about.css, and about hero section
Add the previous summits section to the about page
Add the partners' section to the about page
Add the footer section to the about page
Add a desktop view for the navbar
Add a desktop view for the hero section of the home page
Add a desktop view for the program section of the home page
Add a desktop view for the speakers' section of the home page
Add a desktop view for the partners' section of the home page
Add a desktop view for the footer of the home page
Add a desktop view of the navbar to the about page
Add a desktop view for the hero section of the about page
Add a desktop view for the logo section of the about page
Add a desktop view for the previous section of the about page
Add a desktop view for the partners' section of the about page
Add a desktop view for the footer section of the about page
Add JS mobile menu link handler
Load speaker data with JS
Update the hero header text with a background image over the text
Update the readme and MIT license files
Add an updated video of the project highlighting the code of select features:
[https://www.loom.com/share/0c11d401cb6e43bc88c40d42435a6915](https://www.loom.com/share/0c11d401cb6e43bc88c40d42435a6915?fbclid=IwAR0IGsw4xPR7Z2RKJ_i-WmXQdXisSqeZfuDT_5T45PfPZRvPEn_X8XcxC2w)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,10 +5,209 @@
...
+      </nav>
+    </header>
+
+    <section>
+      <div id="mobile-menu" class="d-none flex-column">
+        <header>
+          <div id="close-menu-btn">
+            <i class="fa-solid fa-x"></i>
+          </div>
+        </header>
```
Comment (binhussen) [inline index.html]: - Kindly keep the semantic flows `header` representing your heading part, not a specific header.👍👌
- I think it is better to rap the `section`'s with `main` for more readability 👍

File: index.html [hunk 2]
```diff
@@ -5,10 +5,209 @@
...
+
+    <section id="home-partners">
+      <h2>Partners</h2>
+      <div id="home-partners-bar"></div>
+      <div id="home-partners-logos">
+        <img src="./assets/images/mozilla.png" />
+        <img src="./assets/images/google.png" />
+        <img src="./assets/images/naver.png" />
+        <img src="./assets/images/daumkakaologo.png" />
+        <img src="./assets/images/airbnb.png" />
```
Comment (binhussen) [inline index.html]: - Kindly use `alt` for each of your images for readability and accessibility👌👍

File: README.md [hunk 3]
```diff
@@ -60,162 +32,107 @@ After you're finished please remove all the comments and instructions!
...
-- **[key_feature_3]**
+- **Responsive Pages**
+- **Mobile Menu interactions using JS**
+- **Load webpage data using JS**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
```
Comment (binhussen) [inline README.md:59]: - Kindly Add a video presentation in the code review summary which includes the features of the project within the maximum length of the video is 5 minutes. 👌👍

File: about.html [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+
+    <footer id="mobile-footer">
+      <img src="./assets/images/rails.png" />
+      <p>
+        <i class="fa-solid fa-copyright"></i> 2023 Aruba Ruby on Rails. Some
+        Rights Reserved.
+      </p>
+    </footer>
+
+    <footer id="desktop-footer">
```
Comment (binhussen) [inline about.html]: - Kindly collect the footer one `footer` tag, becouse you have only one fooer per page for semantic reasons👌👍

File: assets/stylesheets/index.css [hunk 5]
```diff
@@ -1,3 +1,367 @@
...
+  #footer img {
+    height: 64px;
+  }
+
+  #footer p,
+  #footer span {
+    color: #3b3e45;
+    font-weight: 700;
+    font-size: 10px;
+  }
```
Comment (binhussen) [inline assets/stylesheets/index.css:366]: - [Optional] Kindly use class to make your style reuasable👌👍

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @hafedEfheij , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️


## Suggested changes
- kindly add the video presentation as the [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md)👌👍
- Kindly Keep semantic tags for readability and search engines👌👌

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: about.html [hunk 6]
```diff
@@ -0,0 +1,206 @@
...
+          <h5>Aruba Ruby Conference</h5>
+        </article>
+      </div>
+    </section>
+
+    <section id="about-partners">
+      <h2>Partners</h2>
+      <div id="about-partners-bar"></div>
+      <div id="about-partners-logos">
+        <img src="./assets/images/mozilla.png" />
```
Comment (huseyinbiyik) [inline about.html]: Please add alt attributes for each `` element in your pages as the previous reviewer requested. This is important for the accessibility and SEO of your web pages.

File: assets/stylesheets/about.css [hunk 7]
```diff
@@ -0,0 +1,339 @@
...
+  padding: 0;
+  display: grid;
+  grid-template-rows: 1.5fr 1.5fr 1fr;
+}
+
+#about-hero-bar {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
```
Comment (huseyinbiyik) [inline assets/stylesheets/about.css:16]: [OPTIONAL] This line repeats many times in your CSS files. Kindly follow the mobile-first approach and use the desktop media query once in a file to make your code readable and DRY.

File: index.html [hunk 8]
```diff
@@ -5,10 +5,209 @@
...
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+          <li id="cc_campaign"><a href="#">RoR Campaign</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <section>
+      <div id="mobile-menu" class="d-none flex-column">
```
Comment (huseyinbiyik) [inline index.html:74]: [OPTIONAL] Kindly fix your mobile as seen in the design provided. It needs to be transparent. And also there is a bug when you the page to the bottom. Please check the gif below.
Screen Recording 2023-01-06 at 8 21 13 PM

Cindy's design:
Screenshot 2023-01-06 at 8 24 17 PM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @hafedEfheij,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.
tenor

 ## Highlights
- Good job with the implementation. It looks wonderful 👨‍🎨

 ## Required Changes ♻️
- The presentation video is good. You did a great job but it needs to be a maximum of 5 minutes as the previous reviewer requested.
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @hafedEfheij         :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Great job addressing some of the changes that were requested by a previous reviewer :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️


- [ ] You have done a great job taking the presentation video as requested of you by the previous reviewer. However, in order for it to meet all the project requirements, you are supposed to show:
   -  a description of the project. (This includes the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md) as provided by Microverse)
   - a demo of the project features (Which you did a great job :fire: )
   - you should also highlight some interesting piece of code or something you built that you are very proud of. 

The video currently does not showcase all the above three cases. Also, it is a strict requirement that the video is not more than 5 minutes. Kindly retake the video and this time round make sure all three have been covered.

image




_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by hafedEfheij** [comment]
@Nemwel-Boniface thanks for your review, However, this wasn't mentioned in the previous review, I mean I thought the reviews should be related. I mean that was my third review request. and please check the video again I think I did  describe the project and highlight interesting part , they said max 5 min that is why, I did it in brief .  thanks again

**Comment by Nemwel-Boniface** [comment]
Hello @hafedEfheij the last reviewer @huseyinbiyik mentioned it here

image

**Comment by hafedEfheij** [comment]
@Nemwel-Boniface Thanks for reolying. yes that was the first reviewer because I did not put the link.

**Review by frank1738 — APPROVED** [review]
Hi @hafedEfheij , 

Excellent work. Your project is complete!  There is nothing else to say other than... it's time to merge \
Congratulations! 🎉 





## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ Good Design 👍 
✅ Professional Documentation 👍 

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**